### DISCUSSION TITLE
Setting compression level

### DISCUSSION THREAD

**Discussion by bubelov** [actix/actix-web#2924] [Q&A] [answered]
Is it possible to adjust the compression level in `actix-web`? I'm interested in `brotli` compression in particular, and I didn't find anything on that in the docs nor issues.

**Comment by robjtede** [comment, marked as answer]
No theres no way to set it with the current API. We chose the current levels as a good compromise between speed and compression ratio. Would welcome a feature request issue for this, however.

**Reply by bubelov** [reply]
Done: https://github.com/actix/actix-web/issues/2928
